foundation.md

```diff
@@ -46,6 +82,9 @@ Software Foundation is responsible for reviewing, approving, and tracking all
 signed CLAs. This ensures that all code contributed to Trino is fully under the
 Apache License.
 
+The CLA is identical to the CLA of the Apache Software Foundation. All copyright
+for contributions remains with the individual authors.
+
 ### Ownership and trademarks <a name="tm"></a>
```

Comment:
Need to review https://blog.rust-lang.org/2024/11/06/trademark-update.html for some more help on fleshing this out maybe